src/transformers/__init__.py

```diff
@@ -827,6 +829,16 @@
             "LayoutLMPreTrainedModel",
         ]
     )
+    _import_structure["models.layoutlmv2"].extend(
+        [
+            "LAYOUTLM_V2_PRETRAINED_MODEL_ARCHIVE_LIST",
```

Comment:
There is an inconsistency between `LAYOUTLMV2_PRETRAINED_MODEL_ARCHIVE_LIST` and `LAYOUTLM_V2_PRETRAINED_MODEL_ARCHIVE_LIST` across the contributed code